VPC-Section-Formatting (#141)
C2Devel/terraform-provider-croccloud: docs/data-sources/route.md

Reviewer: ```suggestion
The following example shows how a CIDR value may be used to find the ID of a network interface and read the network interface data using the data source.
```
(с Ритой обсуждали формулировку, может еще докрутить )

Reviewer: м-м, это же data source
when finding -> to find
for example, to find ...

Author: Обезличил (сделал it)

Provides information about a route.

> It can be used to find the resource associated with a CIDR. For example, to find the peering connection associated with a CIDR value.

Reviewer: ```suggestion
The data source can be used to find the resource associated with a CIDR. For example, to find the peering connection associated with a CIDR value.
```

> This data source can be used to find the resource associated with a CIDR. For example, to find the peering connection associated with a CIDR value.

Reviewer 2: Думаю, что можно убрать

}
```

## Argument reference

The arguments of this data source act as filters for querying specific routes in the current region.

~> **Note** The given filters must exactly match the resource whose data will be exported as attributes.

* `route_table_id` - (Required, String) The ID of the specific route table containing the route entry.
* `destination_cidr_block` - (Optional, String) The CIDR block of the route.

> * `gateway_id` - (Optional, String) The ID of the gateway.

Reviewer 2: ```suggestion
* `gateway_id` - (Optional, String) The ID of the internet gateway.
```

}
```

## Argument reference

The arguments of this data source act as filters for querying specific routes in the current region.

> The given filters must exactly match the routes whose data will be exported as attributes.

Reviewer 2: Стандартная формлировка + инфо/варнинг

* `route_table_id` - (Required, String) The ID of the specific route table containing the route entry.
* `destination_cidr_block` - (Optional, String) The CIDR block of the route.
* `gateway_id` - (Optional, String) The ID of the gateway.
* `instance_id` - (Optional, String) The ID of the instance.

> * `network_interface_id` - (Optional, String) The ID of the network interface ID.

Reviewer 2: ```suggestion
* `network_interface_id` - (Optional, String) The ID of the network interface.
```

* `transit_gateway_id` - (Optional, String) The ID of the transit gateway.

## Attribute reference

> All arguments are also exported as result attributes.

Reviewer 2: ```suggestion
All arguments are exported as attributes.
```

## Argument Reference

The arguments of this data source act as filters for querying specific routes in the current region.

> The given filters must match exactly to the routes whose data will be exported as attributes.

Reviewer: тут to не лишнее? match with тогда уж

> The arguments of this data source act as filters for querying the available route in the current region. The given filters must match exactly oneRoute whose data will be exported as attributes.

Reviewer: one route ?
тут точно available ? получается, что в регионе всего один маршрут (matching? specific? desired?)

* `route_table_id` - (Required) (String) The ID of the specific route table containing the route entry.
* `destination_cidr_block` - (Optional) (String) The CIDR block of the route belonging to the route table.
* `gateway_id` - (Optional) (String) The gateway ID of the route belonging to the route table.

> * `instance_id` - (Optional) (String) The ID of the instance to which the route table containing the specific route is associated.

Reviewer: м-м, тут все правильно? Обычные таблицы маршрутизации могут ассоциироваться с подсетями, но не экземплярами

> * `network_interface_id` - (Optional) (String) The Network interface ID of the route belonging to the route table.

Reviewer: network
